cli/fathom_web/commands/serve.py

```diff
@@ -4,18 +4,20 @@
 import pathlib
 import ssl
 
-from click import argument, command, option, Path
+from click import command, option, Path
 
 
 @command()
 @option('--port', '-p', type=int, default=8000,
         help='The port to use (default: 8000)')
-@argument('directory', type=Path(exists=True, file_okay=False))
+@option('--directory', '-d', type=Path(exists=True, file_okay=False), default=os.getcwd(),
+        help='The directory to serve files from (default: current working directory')
 def main(directory, port):
     """
     Serves the files in DIRECTORY over a local https server: https://localhost:PORT.
 
     The default PORT is 8000.
```

Comment:
I don't think we need to repeat the defaults here (where they will probably fall out of date anyway) when `--help` auto-formats them from the help strings.

Reply:
That is a good point.